pkg/nsx/services/common/store.go

```diff
@@ -62,6 +62,33 @@ func DecrementPageSize(pageSize *int64) {
 	}
 }
 
+func (resourceStore *ResourceStore) ListIndexFuncValues(key string) sets.String {
+	values := sets.NewString()
+	entities := resourceStore.Indexer.ListIndexFuncValues(key)
+	for _, entity := range entities {
+		values.Insert(entity)
+	}
+	return values
+}
+
+func (resourceStore *ResourceStore) GetByKey(key string) interface{} {
```

Comment:
We can add some comments for GetByKey and GeyByIndex functions， it's helpful to understand the usage of these two functions.

Reply:
ok